### PR TITLE
Update feature-release.md

### PR DESCRIPTION
**Pull request by amyblais**
Editing checklist tasks.
Also combining RC1 and RC testing into one checklist since we don't have self-managed non-QA RC tests anymore. I added a checklist item in the Mobile Release Playbook for Mobile RC testing.

### CODE REVIEW

File: operations/research-and-development/product/release-process/feature-release.md [hunk 1]
```diff
@@ -80,53 +81,37 @@ Day when Leads and PMs decide which major features are included in the release,
...
-3. QA:
+    - Generate an E20 5000 seat test licence and email to Lindy for release testing
+4. QA:
     - Confirm up to date with testing PRs and resolved tickets
     - Confirm up to date with test updates and known issues in release test management and automated tests
     - Assign release testing areas to team members
     - After RC1 is cut: Update test server invite links in Release Testing instructions
     - After RC1 is cut: Run automated regression tests
-4. Build:
+    - Post reminders in Release Announcements channel ([template](https://docs.google.com/document/d/1UvTsvwZXmEL9QPjxmjoIkR2AcwGtOjql8cYRfk2N8eA/edit#bookmark=id.pirw51cwsja5)) for Mobile App RC testing
```
Comment (lindalumitchell) [inline operations/research-and-development/product/release-process/feature-release.md]:
```suggestion
    - Post reminder in Release Announcements channel for any unfinished RC testing, at-mentioning tester(s)
```

File: operations/research-and-development/product/release-process/feature-release.md [hunk 2]
```diff
@@ -80,53 +81,37 @@ Day when Leads and PMs decide which major features are included in the release,
...
+       - Find QA or other teammates to help finish unfinished tests if needed
+       - Verify all release tests are finished, bring any concerns to Triage/Release meeting
+       - Go through all tabs of testing spreadsheet and verify all comments and questions have been filed in Jira as needed
```
Comment (lindalumitchell) [inline operations/research-and-development/product/release-process/feature-release.md]:
```suggestion
       - Go through test cycles and verify all issues and comments have been filed or updated in Jira as needed
```

**Review by lindalumitchell — DISMISSED** [review]
Great updates, Amy; just a couple minor suggestions.

**Review by lindalumitchell — APPROVED** [review]
Looks great, thanks Amy!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**